packages/middleware-sdk-s3-control/src/validate-account-id.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  HttpResponse,
+  InitializeHandler,
+  InitializeMiddleware,
+  InitializeHandlerArguments,
+  InitializeHandlerOptions,
+  InitializeHandlerOutput,
+  MetadataBearer,
+  Pluggable
+} from "@aws-sdk/types";
+
+export function validateAccountIdMiddleware(): InitializeMiddleware<any, any> {
```

Comment:
Validating account id doesn't need to be a middleware, it's only needed in prependAccountId middleware IMO. You can move this logic into the other middleware.

Reply:
removed this separate middleware in 5f4a8455ce86739a463ea614b0afaea0c4cf0ea7

Combined the validation into the prepend middleware